File: Algorithms/Implementation/Bon Appetit/bonAppetit.cs
//Problem URL: https://www.hackerrank.com/challenges/bon-appetit/problem

static void bonAppetit(List<int> bill, int k, int b) {
    var otherItems = bill.Where((x, y) => y != k);

    int total = 0;
    foreach(int price in otherItems){
        total += price;
    }

    if((total / 2) == b)
        Console.Write("Bon Appetit");
    else
        Console.Write(b - (total / 2));
}

File: Algorithms/Implementation/Bon Appetit/bonAppetit.py
#Problem URL: https://www.hackerrank.com/challenges/bon-appetit/problem

def bonAppetit(bill, k, b):
    bill.pop(k)

    total = 0
    for price in bill:
        total += price
    
    if((total / 2) == b):
        print("Bon Appetit")
    else:
        print(int(b - (total / 2)))
